src/components/PersonalDetails.js

```diff
@@ -107,10 +106,15 @@ const PersonalDetails = ({ nextFormStep }) => {
                                 classNamePrefix="select"
                                 className="text-black max-w-3xl"
                                 value={
-                                    profileTagsChoices.filter(option => value.includes(option.value))
+                                    value.map(v => {
```

Comment:
hey @AurelliaChristie can you explain in simple terms what these 6 lines of code is checking for? :)